arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/AccountSection.java

```diff
@@ -46,6 +48,9 @@ public AccountSection(Hub hub) {
     if (hub.opCode().isAnyOf(OpCode.SELFBALANCE, OpCode.CODESIZE)) {
       if (Exceptions.any(exceptions)) {
         // the "squash parent return data" context row is all there is
+        Preconditions.checkArgument(Exceptions.outOfGasException(exceptions));
+        // NOTE: no other exception beyond OUT_OF_GAS_EXCEPTION are supposed to be managed here
+        commonValues.setTracedException(TracedException.OUT_OF_GAS_EXCEPTION);
         return;
       }
```

Comment:
Same issue here as in tackOnlySection : in case of exception you don't trace anything. Can you check for all sections that yih didn't introduce too early return plz ?

Reply:
This was already like this before I added the `tracedException` @letypequividelespoubelles so, if it is an issue, we have it also in other branches.

Reply:
In general, I did not add any return except for `StackOnlyFragment`, that is now fixed.

Reply:
You're right, my bad 👌